package.json

```diff
@@ -131,6 +132,7 @@
     "redux-persist": "^6.0.0",
     "regexparam": "1.3.0",
     "rimraf": "3.0.2",
+    "rooks": "^7.1.2",
```

Comment:
use the hooks that we have.  
if you want to change them - lets do this everywhere